authserver/api/static/css/style.css

```diff
@@ -1,51 +1,80 @@
-html,
-body {
-  height: 100%;
+:root {
+  --blue: #0fbaa8;
+  --light-blue: #0fbaa8b3;
+  --gray20: #333333;
 }
 
 body {
-  display: -ms-flexbox;
-  display: -webkit-box;
+  font-family: "Montserrat", sans-serif;
+  font-size: 18px;
+}
+
+.login {
   display: flex;
-  padding-bottom: 40px;
-  background-color: #333333;
-  font-family: "Raleway", sans-serif;
+  /* "column" – major axis is vertical */
+  flex-direction: column;
+  /* controls layout of the major axis (vertical) */
+  justify-content: center;
+  /* centers all the children on the minor axis (horizontal) */
+  align-items: center;
+  height: 100vh;
 }
 
 label {
-  color: #ffffff;
+  color: #000000;
   text-align: left;
-  padding-top: 25px;
-  font-size: larger;
+  margin-bottom: 0.4rem;
+  float: left;
 }
 
 label.form-check-label {
   padding-top: 0;
   color: inherit;
-  font-size: large;
 }
 
-.form-control {
-  border-radius: 0;
-  border: 0;
+.form-check-inline .form-check-input {
+  margin-top: 0.3rem;
 }
+
+.form-check-inline {
+  display: flex;
+  padding-left: 0;
+  margin-right: 0.75rem;
+  align-items: flex-start;
+}
+
+.form-consent {
```

Comment:
you can save a block and just do `.form-consent, .form-signin { ` or just have one class `.form` 

grouping like styles makes it easier to update if needed

Reply:
done!

Reply:
https://github.com/brighthive/authserver/pull/40/commits/058e0b3b74e80c09a6ab13621a8e69cc93116555#diff-4e97e9f1501c1db50a046e31cb5d11654406f9785feedcc2a051f01cbd0df56aR32